chart/values.yaml

```
@@ -4,25 +4,13 @@ image:
   tag: ""                                    # Container Tag
 service:
   enabled: true                              # Creates a Kubernetes Service for the keptn-service-template-go
-
-distributor:
-  stageFilter: ""                            # Sets the stage this helm service belongs to
-  serviceFilter: ""                          # Sets the service this helm service belongs to
-  projectFilter: ""                          # Sets the project this helm service belongs to
   pubsubTopic: "sh.keptn.>"                  # Sets the events the service subscribes to
```

I don't think this is well placed at `service.pubsubTopic`.

We could do
```suggestion
subscription:
  pubstucTopic: "sh.keptn.>"
```
what do you think?

---

Yes `subscription` definitely makes more sense.